src/frontend/src/components/NavBarV2/index.tsx

```diff
@@ -0,0 +1,184 @@
+import React, { useState } from "react";
+import ENV from "src/common/constants/ENV";
+import Exclamation from "src/common/icons/IconExclamation.svg";
+import CloseIcon from "src/common/images/close-icon.svg";
+import HeaderLogo from "src/common/images/gen-epi-logo.svg";
+import { useUserInfo } from "src/common/queries/auth";
+import { ROUTES } from "src/common/routes";
+import UserMenu from "./components/UserMenu";
+import {
+  AnnouncementBanner,
+  AnnouncementText,
+  AnnouncementTextBold,
+  Bar,
+  ButtonLink,
+  HeaderContainer,
+  HeaderLogoContainer,
+  HeaderMaxWidthContainer,
+  HeaderTopContainer,
+  HeaderTopLinks,
+  MobileNavClose,
+  MobileNavCloseContainer,
+  MobileNavLink,
+  MobileNavLinkContainer,
+  MobileNavSeparator,
+  MobileNavToggle,
+  MobileNavTray,
+  OrgSplash,
+  TextLink,
+} from "./style";
+
+export default function NavBarV2(): JSX.Element {
+  const [menuOpen, setMenuOpen] = useState(false);
+
+  function toggleMobileNav() {
+    setMenuOpen(!menuOpen);
+  }
+
+  const { API_URL } = ENV;
+
+  const { data } = useUserInfo();
+
+  const group = data?.group;
```

Comment:
With this line and line 56, you can quickly unpack using something like
`const { group, user } = data;`